CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -51,7 +51,7 @@ private extension MyCoinView {
             // TODO: - Remove dummy
             TimeFrameSection()
             CoinStatisticsSection(
-                coinDetailItem: CoinDetailItem(
+                coinDetailItem: viewModel.coinDetail ?? CoinDetailItem(
                     coinDetail: MockCoinDetail.single
                 )
```

Comment:
Since we are using real data now, we can also remove the TODO, right?

Reply:
Good catch 👍, the latest commit has already removed it after I rebased :D.